cmd/chunks-inspect/README.md

```diff
@@ -0,0 +1,4 @@
+# Tools for inspecting Loki chunks
+
+This tool can parse Loki chunks and print details from them. Useful for Loki developers.
+
```

Comment:
Can you give a quick example how to install and use ?

Reply:
Good idea, I will do that.